.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: PR - Run Tests
+on:
+  pull_request:
+
+env:
+  GIT_LOCAL_BRANCH: ${{ github.sha }}
+
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      # - name: env
```

Comment:
Do we have a ticket to re-enable tests? (Just for the backlog)

Reply:
https://bcdevex.atlassian.net/browse/CCFPCM-395